app/models/pull_request.rb

```diff
@@ -96,11 +96,15 @@ def ensure_label_is_detached(label)
   ##
   #  Add a comment with the given text
   def add_comment(text)
+    # TODO: why can request be nil and what is request
     req = begin
             request
           rescue StandardError
             nil
           end
+    # Only attach a comment if eligible_for_comment is true (The first iteration after the mergeable state changed to false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long. [124/100]](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5451518)